pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta:__legacy__"
```

Comment:
The `__legacy__` thing is not supposed to be used. It is a fallback for when people forget to specify the `build-backend` entry at all.
It should just be `build-backend = "setuptools.build_meta"`: https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html.